_includes/content/ctl-container-install.md

```diff
@@ -55,7 +55,7 @@ kns.kube-system      kns.kube-system
 We recommend setting an alias as follows.
 
 ```
-alias calicoctl="kubectl exec -i -n kube-system calicoctl /calicoctl -- "
+alias calicoctl="kubectl exec -i -n kube-system calicoctl -- /calicoctl --"
```

Comment:
This needs to be removed. Tested and this alias is correct:

alias calicoctl="kubectl exec -i -n kube-system calicoctl -- /calicoctl"